minitest/utils/jwt/generate_token.py

```
@@ -0,0 +1,26 @@
+import jwt
+from datetime import datetime, timedelta
+from django.conf import settings
+
+def create_jwt_token(user):
+    """
+    사용자 정보를 기반 JWT 토큰 생성
+    """
+    now = datetime.utcnow()
+    expiration_time = now + timedelta(hours=5)
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)